plugin/storage/integration/elasticsearch_test.go

```diff
@@ -100,27 +101,26 @@ func (s *ESStorageIntegration) getVersion() (uint, error) {
 	return uint(esVersion), nil
 }
 
-func (s *ESStorageIntegration) initializeES(allTagsAsFields, archive bool) error {
+func (s *ESStorageIntegration) initializeES(allTagsAsFields, archive bool, t *testing.T) error {
 	rawClient, err := elastic.NewClient(
 		elastic.SetURL(queryURL),
 		elastic.SetSniff(false))
-	if err != nil {
-		return err
-	}
+	require.NoError(t, err)
 	s.logger, _ = testutils.NewLogger()
 
 	s.client = rawClient
 	s.v8Client, err = elasticsearch8.NewClient(elasticsearch8.Config{
 		Addresses:            []string{queryURL},
 		DiscoverNodesOnStart: false,
 	})
-	if err != nil {
-		return err
-	}
+	require.NoError(t, err)
+
+	err = s.initSpanstore(allTagsAsFields, archive)
+	require.NoError(t, err)
```

Comment:
```suggestion
	s.initSpanstore(t, allTagsAsFields, archive)
```
and let it check the error internally